ibm/mas_devops/roles/cp4d_service/defaults/main.yml

```diff
@@ -9,7 +9,7 @@
 # from the IBM operator catalog at present, much more work is needed in this collection to support the operating
 # model of the Cloud Pak for Data team :/
 cpd_product_version: "{{ lookup('env', 'CPD_PRODUCT_VERSION') }}"
-cpd_supported_versions: ['4.0.9', '4.5.0', '4.5.1', '4.5.2', '4.6.0']
+cpd_supported_versions: ['4.0.9', '4.5.0', '4.5.3', '4.6.0', '4.6.3']
```

Comment:
As with previous comment, let's kill 4.0.9 support in this update, it's not a viable install option any more due to CPD support model.